corehq/form_processor/models.py

```diff
@@ -88,4 +124,15 @@ def read_content(self):
             content = f.read()
         return content
 
+
+class XFormOperationSQL(models.Model):
```

Comment:
interesting article and will probably be useful later. i'm hesitant to add it into the actual model. i don't think this is used very often and i'm of the opinion that we should keep XFormInstance as trim as possible for faster form processing